eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -155,8 +155,12 @@ jobs:
 
         - template: /eng/common/pipelines/templates/steps/verify-links.yml
           parameters:
-            Directory: ''
-            CheckLinkGuidance: $true
+            ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+              Directory: ''
+              Urls: (eng/common/scripts/get-markdown-files-from-changed-files.ps1)
```

Comment:
Only markdown files? What about links in other places, like source files?

Reply:
This is literally cut&pasted from the P0 language repositories, since the point of this PR is to follow the pattern set by the P0 languages.

We can absolutely build a similar script for other extensions (.cpp .hpp) if it was desired.